### DISCUSSION TITLE
Getting Started / How to Enter Values - Initial Setup (VERY basic question)

### DISCUSSION THREAD

**Discussion by chrisbellco** [basbruss/adaptive-cover#294] [Q&A]
The good news:
I have installed the integration using HACS.
HACS UI shows it is downloaded.
HA/Settings/Devices shows Adaptive Cover as a device.

My problem:
I am unclear how to access any ui or settings for the Adaptive-Cover integration.
Is there a UI to access? If so, where?
I will need to enter my windows and values, but i cannot determine where to go to do so.

Thank you.

**Comment by homeontherocks** [comment]
Did you add the integration via HA->Settings->Integrations (big blue button Add Integration in the lower right of the Integrations screen)?  If so, you should see Adaptive Cover as a "Configured Integration" and can click on it to set the parameters.

**Comment by kamahat** [comment]
> The good news: I have installed the integration using HACS. HACS UI shows it is downloaded. HA/Settings/Devices shows Adaptive Cover as a device.
> 
> My problem: I am unclear how to access any ui or settings for the Adaptive-Cover integration. Is there a UI to access? If so, where? I will need to enter my windows and values, but i cannot determine where to go to do so.
> 
> Thank you.

this video could help : [https://youtu.be/267yi6AN0P4?feature=shared](url)
